xml/art_jeos_quickstart.xml

```diff
@@ -301,8 +301,145 @@
    <package>cloud-init-config-suse</package>, and
    <package>xfsprogs</package>.
   </para>
- </sect2>
+  <sect3 xml:id="sec-cloud-init-intro">
+  <title>cloud-init basics</title>
+  <para>
+  cloud-init applies automatically custom configuration to running instances.
```

Comment:
```suggestion
  <command>cloud-init</command> automatically applies custom configuration to running instances.
```

Reply:
The formatting is only a suggestion.

Running instances of what? What is an instance in this context?